skimage/measure/_regionprops.py

```diff
@@ -281,6 +282,32 @@ def _inertia_eigvals_to_axes_lengths_3D(inertia_tensor_eigvals):
         axis_lengths.append(sqrt(10 * w))
     return axis_lengths
 
+def _check_spacing(spacing, ndims:int):
+    """Checks the spacing parameter
+
+    The `spacing` parameter should be a tuple.
+    If the spacing is a single number (e.g. float)
+    then assume spacing is same in all dimensions.
+
+    Parameters
+    ---------
+    spacing: The spacing of the pixels in the image
+
+    Returns
+    -------
+    Corrected spacing as tuple (if needed)
+    """
+
+    if isinstance(spacing, tuple):
```

Comment:
This gets to be quite a lot, so perhaps simpler would be to do `isinstance(spacing, (tuple, List)` and, if not, build up spacing as you do. Then, check the elements of spacing with `[np.isreal(x) for x in spacing]`.

Reply:
Sure. I'll change it now.